Makefile

```diff
@@ -29,3 +29,17 @@ serve:
 
 gh-deploy:
 	$(RUN) mkdocs gh-deploy
+
+examples/output/single_examples.yaml: examples/input/schema.tsv examples/input/prefixes.tsv examples/input/single_examples.tsv
```

Comment:
ideally this would be driven by a python test - I like makefiles but here it's good to have a standard pattern - everything in output is made by a test_*py file

Reply:
I'll start on a test now.